IntegrationTests/Tests/IntegrationTests/SwiftPMTests.swift

```diff
@@ -69,6 +69,8 @@ final class SwiftPMTests: XCTestCase {
             }
         }
 
+        #if !os(Windows)
+        // SWBINTTODO: Windows fails to link this library package due to a "lld-link: error: subsystem must be defined" error. See https://github.com/swiftlang/swift-build/issues/310
```

Comment:
suggestion: Instead of compile this out of the test, can we move this to it's own test and mark it skipped on windows?

Reply:
Let's do that in a separate change.